robottelo/vm_capsule.py

```diff
@@ -231,12 +231,17 @@ def _setup_capsule(self):
         )
         self.configure_rhel_repo(settings.__dict__[self.distro + '_repo'])
         self.run('yum repolist')
+        self.run('yum -y update')
         self.run('yum -y install satellite-capsule', timeout=1200)
         result = self.run('rpm -q satellite-capsule')
         if result.return_code != 0:
             raise CapsuleVirtualMachineError(
                 'Failed to install satellite-capsule package\n{}'.format(result.stderr)
             )
+        ssh.command(
+            'hammer setting set --name http_proxy_except_list --value [{0}]'.format(self.hostname),
+            hostname=settings.server.hostname,
+        )
```

Comment:
@vsedmik I looked into this and noticed that we can import Settings from robottelo.cli.settings here as well, so we can change code to:

```
result = Settings.list({"search": 'name=http_proxy_except_list'})[0]
if result["value"] == "[]":
    except_list = '[{0}]'.format(self.hostname)
else:
    except_list = result["value"][:-1] + ', {0}]'.format(self.hostname)
Settings.set({'name': 'http_proxy_except_list', 'value': except_list})
```